curriculum/challenges/english/03-front-end-libraries/jquery/change-text-inside-an-element-using-jquery.english.md

```diff
@@ -14,7 +14,7 @@ Here's how you would rewrite and emphasize the text of our heading:
 jQuery also has a similar function called <code>.text()</code> that only alters text without adding tags. In other words, this function will not evaluate any HTML tags passed to it, but will instead treat it as the text you want to replace the existing content with.
 Change the button with id <code>target4</code> by emphasizing its text.
 <a href="https://developer.mozilla.org/en/docs/Web/HTML/Element/em" target="_blank">View the MDN web docs for &#60;em&#62;</a> to learn the difference between <code>&#60;i&#62;</code> and <code>&#60;em&#62</code> and their uses.
-Note that while the <code>&#60;i&#62;</code> tag has traditionally been used to emphasize text, it has since been coopted for use as a tag for icons. The <code>&#60;em&#62;</code> tag is now widely accepted as the tag for emphasis. Either will work for this challenge.
+Note that while the <code>&#60;i&#62;</code> tag has traditionally been used to emphasize text, it has since been co-opted for use as a tag for icons. The <code>&#60;em&#62;</code> tag is now widely accepted as the tag for emphasis. Either will work for this challenge.
```

Comment:
@csteacher1, @thecodingaviator I have never liked the word `co-opted` here.  What do you think about the following?
```suggestion
Note that while the <code>&#60;i&#62;</code> tag has traditionally been used to emphasize text, it has since been adopted for use as a tag for icons. The <code>&#60;em&#62;</code> tag is now widely accepted as the tag for emphasis. Either will work for this challenge.
```

Reply:
LGTM too

Reply:
I like it!

Reply:
@csteacher1 Go ahead and click the `Commit suggestion` button above to apply my suggested change.